api/apps/articles/test/model/index/persistence.test.coffee

```diff
@@ -732,3 +732,18 @@ describe 'Article Persistence', ->
         article.hero_section.type.should.equal 'split'
         article.hero_section.deck.should.equal 'This year was incredible.'
         done()
+
+    it 'saves a feature hero_section in basic', (done) ->
+      Article.save {
+        hero_section:
+          url: 'http://image.com'
+          type: 'basic'
+          deck: 'This year was incredible.'
+          cover_image_url: 'http://some-cover-image.png'
+      }, 'foo', {}, (err, article) ->
+        return done err if err
+        article.hero_section.url.should.equal 'http://image.com'
+        article.hero_section.type.should.equal 'basic'
+        article.hero_section.deck.should.equal 'This year was incredible.'
+        article.hero_section.cover_image_url.should.equal 'http://some-cover-image.png'
```

Comment:
@kanaabe - for whatever reason this test only fails on Circle -- thoughts? Since I updated the schema do I need to run a migration of some sort?